tgui/packages/tgui/layouts/Window.tsx

```diff
@@ -52,32 +52,34 @@ export const Window = (props: Props) => {
   const { config, suspended } = useBackend();
   const { debugLayout = false } = useDebug();
 
-  useEffect(() => {
-    const updateGeometry = () => {
-      const options = {
-        ...config.window,
-        size: DEFAULT_SIZE,
+  if (!suspended) {
+    useEffect(() => {
```

Comment:
Violates the rule of hooks, hooks cannot be conditional. Could swap this order

https://react.dev/reference/rules/rules-of-hooks

Reply:
This better? Seems to also work.